packages/components/inputs/rich-text-utils/src/plugins/mark.tsx

```diff
@@ -34,21 +67,25 @@ const MarkPlugin = (options = {}) => {
 
   return [
     {
-      // eslint-disable-next-line consistent-return
-      onKeyDown(event, editor, next) {
-        if (!isHotKey(event)) {
+      onKeyDown(
+        event: TEvent,
+        editor: { toggleMark: (arg0: unknown) => void },
+        next: () => void
+      ): void {
+        if (!isHotKey(event as KeyboardEvent)) {
           return next();
         }
 
         event.preventDefault();
         editor.toggleMark(options.typeName);
       },
-      renderMark(props, editor, next) {
+      renderMark(props: TProps, next: () => ReactJSXElement) {
```

Comment:
Here we're changing the function signature and I'm wondering if all consumers have also been adapted to it or checked that it won't affect them.

Reply:
hmm, I get you here. I might just see if I can make it uniform else I would restore to the initial function siignature and use `_` on the `editor` parameter since it's mostly not even used in the function.